frontend/benefit/handler/src/hooks/useHandlerReviewActions.tsx

```diff
@@ -69,7 +68,7 @@ const useHandlerReviewActions = (
     calculator: CalculationCommon
   ): Promise<ApplicationData | void> => {
     try {
-      await updateApplication(getDataEmployment(calculator));
+      void updateApplicationQuery.mutate(getDataEmployment(calculator));
     } catch (error) {
```

Comment:
try-catch does not work, you can just remove it and use onError if needed: `updateApplicationQuery.mutate(getDataEmployment(calculator), {onError: () => { ...}})`